CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ if((NOT LIBOCPP_ENABLE_V16) AND (NOT LIBOCPP_ENABLE_V201))
     message(FATAL_ERROR "At least one of LIBOCPP_ENABLE_V16 and LIBOCPP_ENABLE_V201 needs to be ON")
 endif()
 
+if(LIBOCPP_ENABLE_DEPRECATED_WEBSOCKETPP)
+    message(FATAL_ERROR "Websocket++ has been deprecated")
```

Comment:
It's no longer deprecated but its removed now right?

Reply:
I was wondering, if I should even let that inside there. But I thought that if someone has a config that has that option on, maybe a error should be issued.

Reply:
Yeah I understand keeping the fatal error, I would just change the message a bit since deprecated means you can still use it, and you can't use this anymore.

Reply:
Changed it to 'removed' from 'deprecated', should be more clear.